activemodel/lib/active_model/secure_password.rb

```diff
@@ -111,8 +113,8 @@ def has_secure_password(attribute = :password, validations: true)
             end
           end
 
-          validates_length_of attribute, maximum: ActiveModel::SecurePassword::MAX_PASSWORD_LENGTH_ALLOWED
-          validates_confirmation_of attribute, allow_blank: true
+          validates_length_of attribute, maximum: ActiveModel::SecurePassword::MAX_PASSWORD_LENGTH_ALLOWED, **validation_options
+          validates_confirmation_of attribute, allow_blank: true, **validation_options
```

Comment:
> On the last 2 lines you changed, does it matter to have the `**validation_options` at the end of the line rather than before the other rules ?

The difference between the two is that if `validation_options` has `:maximum` or `:allow_blank` keys, it will override those values for `validates_length_of` and `validates_confirmation_of`.  It's probably fine either way, but, just in case someone re-uses an options hash (which would be a semi-plausible way to reuse `:if` / `:unless` Procs), putting `**validation_options` first would be safer.

Reply:
Hmm, I can see both points to be valid points (one being safer as you say). On the other hand, if you _did_ want to set `:allow_blank` it might not be obvious that this is overwritten and therefore you might be left wondering why it doesn't work. 
I am perhaps over thinking things here though. 
I will happily make that change